aztec/src/main/kotlin/org/wordpress/aztec/watchers/event/sequence/ObservationQueue.kt

```diff
@@ -0,0 +1,65 @@
+package org.wordpress.aztec.watchers.event.sequence
+
+import android.os.Build
+import org.wordpress.aztec.watchers.event.IEventInjector
+import org.wordpress.aztec.watchers.event.buckets.API26Bucket
+import org.wordpress.aztec.watchers.event.text.TextWatcherEvent
+import org.wordpress.aztec.watchers.event.buckets.Bucket
+
+class ObservationQueue(injector: IEventInjector) : EventSequence<TextWatcherEvent>() {
+    val buckets = ArrayList<Bucket>()
+    val injector = injector
```

Comment:
This can be moved to the constructors.

Reply:
addressed the various inspector suggestions (only those that I considered applicable or made sense) in 0984aa0